### ISSUE TITLE
Model compile fails with uniform RV

### ISSUE THREAD

**Issue by fonnesbeck** [pymc-devs/nutpie#19]
Running nutpie 0.4.0 on macOS, I get compile failures when trying to compile a model with a uniform RV (or presumably any interval-constrained RV):

```python
with pm.Model() as mod:
    rho = pm.Uniform("rho", -1, 1)
```

```python
compiled_model = nutpie.compile_pymc_model(mod)
```

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/phillies/pie/research/projections/pitchers/pitcher_proj.py in <module>
----> 1 compiled_model = nutpie.compile_pymc_model(mod)

~/GitHub/nutpie/nutpie/compile_pymc.py in compile_pymc_model(model, **kwargs)
    119     """Compile necessary functions for sampling a pymc model."""
    120 
--> 121     n_dim, logp_fn_at, logp_fn, expand_fn, shared_expand, shape_info = _make_functions(
    122         model
    123     )

~/GitHub/nutpie/nutpie/compile_pymc.py in _make_functions(model)
    212 
    213         joined_names.append(value_var.name)
--> 214         shape = shapes[value_var.name]
    215         joined_shapes.append(shape)
    216         length = prod(shape)

KeyError: 'rho_interval__'
```

**Comment by bwengals** [comment]
bumped into this too!

**Comment by aseyboldt** [comment]
Should be fixed by #20, with dev version of pymc and pytensor.
